code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -134,7 +134,7 @@
 	name = "Heretics"
 	antag_flag = ROLE_HERETIC
 	antag_datum = /datum/antagonist/heretic
-	protected_roles = list("Prisoner","Security Officer", "Warden", "Detective", "Head of Security", "Captain")
+	protected_roles = list("Security Officer", "Warden", "Detective", "Head of Security", "Captain", "Brig Physician")
```

Comment:
I think you forgot deputy.

Reply:
deputy is not a roundstart role, so this doesnt matter

Reply:
Doesn't deputy show up in the job selection list and on join list if HOP lets them spawn?

Reply:
Deputy isn't on any protected roles lists for any other antag.

Reply:
My bad then

Reply:
What about chaplain though? He's not supposed to be cultist so why should he be the heretic?